CMakeLists.txt

```diff
@@ -504,6 +505,21 @@ install(
     ${INSTALL_CONFIGDIR}
 )
 
+if (${TRITON_PYTORCH_ENABLE_PYTHON_RUNTIME})
```

Comment:
Does this mean that we'll include the runtime by default? I believe the environment size could be significant since it contains CUDA, PyTorch, etc.

Reply:
Yes, but the behavior can be easily changed. The current size of the conda pack is 3.00 GB.